CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 ## 3.0.1
+### Added
```

Comment:
Move this up a line to the Unreleased section. Right now the change is listed in the previous release